src/main/kotlin/lotto/WinningNumbers.kt

```diff
@@ -0,0 +1,12 @@
+package lotto
+
+data class WinningNumbers(val winningNumbers: Lotto, val bonusNumber: Int) {
+    init {
+        require(bonusNumber in Lotto.MIN..Lotto.MAX) {
+            throw IllegalArgumentException()
+        }
+        require(!winningNumbers.numbers.contains(bonusNumber)) {
+            throw IllegalArgumentException()
+        }
+    }
```

Comment:
check `require` & require kind message 🤣 

I think the `LottoNumber` and `BonusNumber` are the same thing.
How about wrapping both of them with the same VO (Value Object) and moving the validation there?